docs/05-gitlab/01-getting-started.mdx

```diff
@@ -22,55 +21,109 @@ Any subsequent steps assume you have read the above.
-test and build projects.
+The [unity3d-gitlab-ci-example project](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/) uses
+[unity3d docker images](https://github.com/game-ci/docker/) published by
+[game-ci](https://github.com/game-ci). You can find all the supported Unity versions in our
+[Docker > Versions page](/docs/docker/versions).
 
-It's generally considered good practice to use the same Unity version for your CI/CD setup as you do
-to develop your project.
+It is generally considered good practice to use the same Unity version for your CI/CD setup as you
+do to develop your project. Our Gitlab CI configuration will automatically detect and use the
+correct Unity version for your project.
 
 ## Video tutorial
 
 <Video url="https://www.youtube-nocookie.com/embed/k0NcedDzEqA" />
 
-## Setting up gitlab-ci for your Unity project
+:::caution
 
-### I don't have a Unity project yet
+The video is slightly outdated, but the general concepts are still the same. 👍
 
-1. Fork
-   [the unity3d-gitlab-ci-example project](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/)
-1. Clone the fork you just created locally
-1. Continue to activation instructions
+- Activation process has changed a bit since then.
+- Docker images have been updated
+- The `.gitlab-ci.yml` file uses different variables now.
 
-### I already have my own Unity project
+:::
 
-1. Clone
-   [the unity3d-gitlab-ci-example project](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/)
-1. Copy
-   [`.gitlab-ci.yml`](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/-/blob/main/.gitlab-ci.yml)
-   to the root of your repository,
-   [`Assets/Scripts/Editor/BuildCommand.cs`](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/-/blob/main/Assets/Scripts/Editor/BuildCommand.cs)
-   and [`ci` folder](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/-/blob/main/ci) to your
-   project:
+## Setting Up GitLab CI for Your Unity Project
 
-Assuming you've cloned the example project in the parent folder of your project, and your Unity
-project is at the root of your repository, execute these commands from the root folder of your
-project:
+Follow these steps to quickly set up GitLab CI for your Unity project.
+
+### 1. Run the Commands
+
+Open a terminal in the **parent folder** where you want to set up your Unity project and run these
+commands one by one. In summary, these commands will:
+
+- Clone the example project
+- Copy the necessary files and folders to your Unity project.
+
+:::info
+
+See [example project's release](https://gitlab.com/game-ci/unity3d-gitlab-ci-example/-/releases) for
+stable versions.
+
+:::
+
+Note: This setup assumes that your unity project is at the root of the repository. If your project
+is in a subfolder, you will need to adjust the paths in the commands below. You will also need to
+adjust `UNITY_DIR` in `.gitlab-ci.yml` to point to the correct directory.
 
 ```bash
+# Clone the example repository
```

Comment:
Consider splitting up the code block and put the comments in between them as normal text.